Makefile

```diff
@@ -1,4 +1,4 @@
-CC=gcc
+CC?=gcc
```

Comment:
@chipitsine Because those projects are much more portable than this one. This one requires gcc specific features and clang just happens to be compatible with gcc.